tests/test_BigWig_remote.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import os
+
+from efs_parser.BigWig import BigWig
+
+__author__ = "jkanche"
+__copyright__ = "jkanche"
+__license__ = "mit"
+
+"""
+    The file test.bigBed and test assertions 
+    come from the pyBigWig library
+"""
+
+#bb = BigWig("./test.bw")
+bb = BigWig("https://obj.umiacs.umd.edu/bigwig-files/39031.bigwig")
+#bb = BigWig("https://obj.umiacs.umd.edu/bigwig-files/ENCFF330GHF.bigBed")
+
+def test_correct_format():
+    assert (bb.header['magic']==2291137574)
+
+def test_header():
+    assert(bb.header == {'magic': 2291137574, 'version': 4, 'zoomLevels': 10, 'chromTreeOffset': 344, 'fullDataOffset': 705, 'fullIndexOffset': 582485344, 'fieldCount': 0, 'definedFieldCount': 0, 'autoSqlOffset': 0, 'totalSummaryOffset': 304, 'uncompressBufSize': 32768})
+
+def test_columns():
+    #bb.header[fieldCount]==0
+    assert(len(bb.columns) == bb.header[fieldCount])
+    assert(bb.columns == ["chr", "start", "end", "score"])
+
+def test_range():
+    res, err = bb.getRange(chr="chr1", start=10000000, end=10020000)
```

Comment:
also here - make sure `res` only contain rows within the given range (chr, start & end)